bot.py

```diff
@@ -112,13 +124,119 @@ def on_message(ws, message):
         if admin_message(response):
             return
 
-    elif "team_join" == response["type"]:
-        if NEWUSERGREETING:
-            new_user_message(response)
-        USER_LIST = sc.api_call(  # noqa: 841
-            "users.list",
-        )
 
+    #TODO: Purge all this shit
+    # elif "team_join" == response["type"]:
+    #     if NEWUSERGREETING:
+    #         new_user_message(response)
+    #     USER_LIST = sc.api_call(  # noqa: 841
+    #         "users.list",
+    #     )
+
+    log(response)   
+
+def log(response):
+    global engine
+    def add_user(r):
+        q = engine.execute("SELECT last_name FROM users WHERE slack_id=%s", r["user"])
```

Comment:
This is obviously a query, but is there no way you could have done this without hardcoding sql?